### DISCUSSION TITLE
Unable see PID/Program name on my terminal, please do the need full

### DISCUSSION THREAD

**Discussion by umamkvdu** [learndevopsonline/learndevopsonline#1171] [Q&A]
Hello,

After executing 'netstat -lntp' on my terminal, I couldn't figure out as is displaying on your terminal as below. Please suggest me I'm missing anything. 

![image](https://github.com/learndevopsonline/learndevopsonline/assets/139033187/42ef1fab-3abc-4728-84b2-caf6cedbba3e)

**Comment by Emmaokoro18** [comment]
I think you have to include sudo before you run the command because you are logged in as centos.
try this and see if it works: sudo netstat -lntp

**Reply by umamkvdu** [reply]
Tried adding sudo prior to that, but still not showing the last column.

![image](https://github.com/learndevopsonline/learndevopsonline/assets/139033187/c3c91d8e-d533-424b-8a41-27762b8915c2)

**Comment by maheshkoheda** [comment]
Hi, Sudo is one thing. Also I observed it need not be in last column it could be in 2nd or 3rd column overlapped with a different PID number  compared to Raghu
